README.md

```diff
@@ -1329,8 +1329,8 @@ Key considerations include:
 - Another path is to
   [split the difference](http://stockoptioncounsel.com/blog/early-expiration-of-startup-stock-options-part-2-the-full-10-year-term-solution/2017/3/28)
   and give extended windows only to longer-term employees.
-- Taken together, it’s clear many employees were not clear on the nuances of this before
-  joining companies in the past, and some
+- Taken together, it’s evident many employees have not always been clear on the nuances of
```

Comment:
Rewording "clear" is a good copy edit.

But remove "always".

Also (unrelated but while editing this): "before joining" -> "when joining"

Reply:
Fixed.